raiden/network/proxies/token_network_registry.py

```diff
@@ -236,3 +236,9 @@ def get_token_network_created(self, to_block: BlockSpecification = "latest") ->
         return self.proxy.contract.functions.token_network_created().call(
             block_identifier=to_block
         )
+
+    def get_max_token_networks(self, to_block: BlockSpecification = "latest") -> int:
```

Comment:
We can try to stop doing the "latest" thing.

Reply:
I'll remove the default value here.  It's a good moment because nobody is using the default value yet.